plugins/user_mapping/src/local_file.hpp

```diff
@@ -0,0 +1,17 @@
+#ifndef IRODS_HTTP_API_LOCAL_FILE_HPP
+#define IRODS_HTTP_API_LOCAL_FILE_HPP
+
+#include <optional>
+
+#include <nlohmann/json.hpp>
+
+struct UserProfile {
+	std::string irods_user_name;
```

Comment:
This only needs to work for users in the local zone, right?

Reply:
hrm, an interesting question.    HTTP API *could* auth a remote ... no, that would be redirected / responsibility of a different server/service/zone...

i think local zone user is correct / plenty.    discush?

Reply:
Right, only local zone users can / should be able to authenticate with iRODS. So, maybe the local zone user is an okay assumption here.

Reply:
> This only needs to work for users in the local zone, right?

Yes. Users must always go through their respective zone to reach remote zones.

Reply:
Cool. Resolving.